pom.xml

```diff
@@ -784,10 +804,20 @@
             <version>2.2.0</version>
         </dependency>
         <!-- Provides annotations like XmlElement indicating how classes should be serialized -->
+        <dependency>
+            <groupId>com.sun.activation</groupId>
+            <artifactId>javax.activation</artifactId>
+            <version>1.2.0</version>
+        </dependency>
```

Comment:
This dependency is part of migration from Java 8 to Java 11, so I will remove it.